build/makelib/image.mk

```diff
@@ -48,7 +48,7 @@ ifeq ($(origin BUILD_REGISTRY), undefined)
 BUILD_REGISTRY := build-$(shell echo $(HOSTNAME)-$(ROOT_DIR) | shasum -a 256 | cut -c1-8)
 endif
 
-MANIFEST_TOOL_VERSION=v1.0.3
+MANIFEST_TOOL_VERSION=v1.0.0
```

Comment:
version downgrade?

Reply:
Somehow v1.0.3 doesn't work properly on my machine. I had to downgrade back to v1.